### PR TITLE
Update blockquote styling

### PR DESCRIPTION
**Pull request by garethwilliamsdesign**
### Trello card

[Trello 5292](https://trello.com/c/PNJVWoZI)

### Context

We want to review the quote component and improve the design if needed, to ensure it is as effective as possible.

### Changes proposed in this pull request

Restyle the blockquote component as per the screenshots attached to the ticket.

### Guidance to review

Check that the blockquote component matches the screenshots attached to the ticket.

### CODE REVIEW

File: app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md
Comment (martyn-w) [inline app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md:30]: "Interaction"

File: app/views/content/a-day-in-the-life-of-a-teacher.md
Comment (gemmadallmandfe) [inline app/views/content/a-day-in-the-life-of-a-teacher.md:24]: @garethwilliamsdesign @martyn-w Can we merge the name / job title onto one line as we do on the SEND page?

Otherwise this page looks much better now!

File: app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md
Comment (gemmadallmandfe) [reply app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md:30]: we need to delete this altogether before this PR is merged!

**Review by gemmadallmandfe — COMMENTED** [review]
@martyn-w A few comments for you!

**Comment by gemmadallmandfe** [comment]
@martyn-w In addition, there are two first person blog posts with pull-out quotes which are a repeat of other text

This makes even less sense with the new quote style - having checked with @MaxineCl, we think it is better to remove them entirely

Could you add this change to the PR?

**https://get-into-teaching-app-review-3815.test.teacherservices.cloud/blog/abigails-career-progression-story**

image

image

image

https://get-into-teaching-app-review-3815.test.teacherservices.cloud/blog/from-teaching-internship-to-assistant-head-of-maths

image

image

image

File: app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb
Comment (gemmadallmandfe) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: Happy to go with whatever @jenhadfield-dfe says (we should be pragmatic and low effort though as we just want to get this out)

Comment (jenhadfield-dfe) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: I think for now it works fine with the pink line and white background. ideally we'd remove the white background in this case, but I think its ok for now

Comment (martyn-w) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: let me see if I can remove the white background with low-ish effort...

Comment (martyn-w) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: This is with the white background in the quote, grey background on the text 
Screenshot 2024-02-08 at 14 58 56

Comment (martyn-w) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: And now with a grey background and a pink line:

Screenshot 2024-02-08 at 15 19 03

Comment (jenhadfield-dfe) [reply app/views/content/welcome/my-journey-into-teaching/_introduction.html.erb:46]: the grey background and pink line looks perfect thanks!

**Comment by gemmadallmandfe** [comment]
Looks great @martyn-w - think we are nearly there

My only comment is on now about quotes with a grey background - they all have a pink line now, which looks slightly odd in some places, eg

**https://get-into-teaching-app-review-3815.test.teacherservices.cloud/landing/food-for-thought**

image


**https://get-into-teaching-app-review-3815.test.teacherservices.cloud/subjects/maths**

image

(they look better on mobile)

In the interests of drawing a line under this, I'm not too concerned about this given that we've agreed to revisit in the nearish future - but happy to defer to whatever @jenhadfield-dfe thinks

**Comment by jenhadfield-dfe** [comment]
hey i think this is fine for now until we re-visit this.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged (and we will revisit after next UR)

**Review by ekumachidi — APPROVED** [review]

**Pull request merged**
